services/src/main/java/org/keycloak/protocol/oidc/AccessTokenIntrospectionProvider.java

```diff
@@ -85,6 +85,13 @@ public Response introspect(String token) {
                         }
                     }
                 }
+
+                if (accessToken.getConfirmation() != null && accessToken.getConfirmation().getKeyThumbprint() != null) {
+                    tokenMetadata.put("token_type", "DPoP");
```

Comment:
Is it possible to use some constant for token_type like we're doing for other things? Maybe `OAuth2Constants` is good class to use it or something similar?

Reply:
Yes, the same above.